lib/utils.js

```diff
@@ -95,3 +95,62 @@ exports.findAvailablePort = function(app){
   });
 
 }
+
+exports.autoStoreData = function (req, res, next) {
```

Comment:
the next line is:

`// Middleware - store any data sent in session`

is that ok, or you want to add to that?